modules/basics/eval.js

```diff
@@ -36,7 +36,7 @@ module.exports = class EvalCommand extends global.utils.baseCommand {
 			if(val instanceof Error) {
 				ctx.send(`Callback error: \`${val}\``);
 			} else {
-				const result = this.makeResultMessages(val, process.hrtime(this.hrStart));
+				const result = this.makeResultMessages(this.lastResult, hrDiff, ctx.args.script, ctx);

```

Comment:
Double facepalm
You weren't supposed to copy-paste the other line
Did you even look at the comment pls

Reply:
yep, I noticed something was off and just spent the time prior to trying to find out what.